Model/Service/Stock/Provider/StockProviderInterface.php

```diff
@@ -77,30 +77,39 @@
 interface StockProviderInterface
 {
     /**
+     * Gets the available quantity for a product in the given website
+     *
      * @param Product $product
      * @param Website $website
      * @return int
      */
     public function getAvailableQuantity(Product $product, Website $website);
 
     /**
+     * Checks the availability for a product in the given website
+     *
      * @param Product $product
      * @param Website $website
      * @return bool
      */
     public function isInStock(Product $product, Website $website);
 
     /**
+     * Returns an array of product ids and quantities in stock for the website
+     * for each product id
+     *
      * @param array $productIds
      * @param Website $website
      * @return array ['productId1' => 4, 'productId2' => 2, ...]
      */
     public function getQuantitiesByIds(array $productIds, Website $website);
 
     /**
+     * Checks and returns which of the given product ids are in stock
+     *
      * @param array $productIds
      * @param Website $website
-     * @return array an array of in stock skus having the product id as key ['1' => 'SKUSTRING-1']
+     * @return array an array of in stock productIds ['1','2', etc.]
      */
-    public function getInStockSkusByIds(array $productIds, Website $website);
+    public function getInStockProductIds(array $productIds, Website $website);
```

Comment:
This is causing problems with MSI module. Needs to be updated in that extension as well

Reply:
Yes, it's here: https://github.com/Nosto/nosto-magento2-msi/pull/9